maps/clarion.dmm

```diff
@@ -14545,10 +14545,7 @@
 /area/station/crew_quarters/bar)
 "azC" = (
 /obj/table/reinforced/auto,
-/obj/item/reagent_containers/food/snacks/cake/chocolate{
-	desc = "Holy shit! This cake probably costs more than the gross domestic product of Bulgaria!";
-	name = "Extravagant Chocolate Gateau"
-	},
+/obj/item/reagent_containers/food/snacks/cake/chocolate/gateau,
```

Comment:
Going to have to get signoff from Dions for Clarion modification

Reply:
i just dm'd them on discord and got the ok (^ - ^)